src/components/paint-controls.js

```diff
@@ -115,22 +118,27 @@ AFRAME.registerComponent('paint-controls', {
       // 3 Strokes to hide
       if (self.system.numberStrokes === 3) {
         var tooltips = Array.prototype.slice.call(document.querySelectorAll('[tooltip]'));
-        var object = { opacity: 1.0 };
-
-        var tween = new AFRAME.TWEEN.Tween(object)
-          .to({opacity: 0.0}, 1000)
-          .onComplete(function () {
+        var animObject = { opacity: 1.0 };
+
+        self.hideTooltips = AFRAME.ANIME({
```

Comment:
Wondering if we can use the [animation component](https://aframe.io/docs/1.3.0/components/animation.html#sidebar) instead of managing ANIME manually

Reply:
Something like 

`toolTip.setAttribute('animation', {dur: 1000, delay: 2000, property: 'opacity', from: 1.0, to: 0.0, startEvents: 'stroke-started'});`

and can do `tooltip.emit('stroke-started');`